test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,11 +36,40 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() throws Exception {
+        // no object
+        assertNoNull();
+
+        // only one object and it is null
+        assertHaveNull((Object) null);
+
+        // only one object and it is not null
+        assertNoNull(1);
+        assertNoNull("");
+        assertNoNull("abc");
+
+        // all objects are not null
+        assertNoNull(1, "abc", "aa");
+        assertNoNull(1, Integer.valueOf(123), "");
+        assertNoNull("cab", 23, "", "ba");
+
+        // some objects are null
+        assertHaveNull(1, null, "aa", null, "abc");
+        assertHaveNull(Integer.valueOf(21), "", null);
+        assertHaveNull(null, "cab", null, 23, "");
+    }
+
+
     private void assertAreUnique(Object... objects) {
         assertTrue(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
 
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    private void assertHaveNull(Object... objects) { assertTrue(Utils.isAnyNull(objects)); }
+
+    private void assertNoNull(Object... objects) { assertFalse(Utils.isAnyNull(objects)); }
```

Comment:
As mentioned in tutorial, there is no need to abstract out `assertTrue` and `assertFalse` as separate methods since they are rather short. Existing tests do that to avoid repeating `Arrays.asList` everywhere. You can use them directly in `isAnyNull`.